app/static/css/_map.scss

```diff
@@ -3,7 +3,9 @@
 .map__container {
     display: flex;
     flex-flow: row wrap;
+    padding: 0 3rem;
```

Comment:
This is purely because it's really hard to scroll past the map without it grabbing focus. I'm going to have to take a good look at the mobile layout for this, or do something that makes the map just not do that.